Bug: Interface 'ICanvasRenderingContext2D' incorrectly extends interface 'CanvasTextDrawingStyles'

### Current Behavior

A simple TypeScript project that only imports `@pixi/core` fails to compile with TypeScript:

```
node_modules/@pixi/settings/lib/ICanvasRenderingContext2D.d.ts:6:18 - error TS2430: Interface 'ICanvasRenderingContext2D' incorrectly extends interface 'CanvasTextDrawingStyles'.
  Property 'letterSpacing' is optional in type 'ICanvasRenderingContext2D' but required in type 'CanvasTextDrawingStyles'.
```

### Expected Behavior

Importing `@pixi/core` should work as expected with the latest `@types/web` package.

### Steps to Reproduce

index.ts
```ts
import * as PIXICore from '@pixi/core';

console.log({ PIXICore });
```

package.json
```json
{
  "name": "pixi-debug",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "dependencies": {
    "@pixi/core": "7.3.1"
  },
  "devDependencies": {
    "@types/css-font-loading-module": "0.0.10",
    "@types/web": "0.0.117",
    "typescript": "5.2.2"
  }
}
```

tsconfig.json
```json
{
    "compilerOptions": {
        "module": "commonjs",
        "lib": [
            "ES2015"
        ],
        "esModuleInterop": true
    },
    "exclude": [
        "node_modules"
    ],
    "include": [
        "src"
    ]
}
```


### Environment

- **`pixi.js` version**: *7.3.1*
- **TypeScript version**: *5.2.2*
- **`@types/web` version**: *0.0.117*


### Possible Solution

Change the `letterSpacing` property to required.

### Additional Information

It would probably be good to test that everything compiles with the latest web types. This was just the first problem one I ran into when setting up a clean test project for reproducing the issue(s).

I am having this problem right now.
`node_modules/pixi.js/lib/environment/canvas/ICanvasRenderingContext2D.d.ts:6:18 - error TS2430: Interface 'ICanvasRenderingContext2D' incorrectly extends interface 'CanvasTextDrawingStyles'.
  Types of property 'letterSpacing' are incompatible.
    Type 'string | undefined' is not assignable to type 'string'.
      Type 'undefined' is not assignable to type 'string'.`

--- a/src/environment/canvas/ICanvasRenderingContext2D.ts
+++ b/src/environment/canvas/ICanvasRenderingContext2D.ts
@@ -18,7 +18,7 @@ export interface ICanvasRenderingContext2D extends
     CanvasDrawImage,
     CanvasImageData,
     CanvasPathDrawingStyles,
-    CanvasTextDrawingStyles,
+    Omit<CanvasTextDrawingStyles, 'letterSpacing'>,
     CanvasPath
 {
     /** creates a pattern using the specified image and repetition. */

